LandSandBoat/server
# [Quest] Test My Mettle Implementation

**_I affirm:_**

## What does this pull request do?

It sure has been easy to get a pair of Power Sandals before this work.  Not really sure if anyone will be happy this will start working.

Does not close a known issue.

## Steps to test these changes

- Meet level, rank, and fame requirements
- Zone to Selbina, !gotoname Devean
- Bet him you can find the sandals
- Zone to Davoi, !gotoname Jar and interact
- Zone to Selbina and either talk or trade with Devean
- Verify bet amount and sandal status (removed or still in storage)
- Repeat (there are a few permutations)

Unrelated to the quest logic
- Go to Davoi and find the pot, wait for it to move on a random game hour
